docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,834 @@
+
+Generating files to run your app on Azure:
+
+  (✓) Done: Generating ./azure.yaml
+  (✓) Done: Generating ./next-steps.md
+
+SUCCESS: Your app is ready for the cloud!
+```
+
+Then, deploy the application by running `azd up`:
+
+```azdeveloper
+$ azd up 
+? Select an Azure Subscription to use: 130. [redacted]
+? Select an Azure location to use: 50. (US) West US 2 (westus2)
+
+Packaging services (azd package)
+
+
+Provisioning Azure resources (azd provision)
+Provisioning Azure resources can take some time.
+
+Subscription: [redacted]
+Location: West US 2
+
+  You can view detailed progress in the Azure Portal:
+  [redacted]
+
+  (✓) Done: Resource group: rg-azfunc-piglatin (967ms)
+  (✓) Done: Container Registry: [redacted] (13.316s)
+  (✓) Done: Log Analytics workspace: [redacted] (16.467s)
+  (✓) Done: Container Apps Environment: [redacted] (1m35.531s)
+  (✓) Done: Storage account: [redacted] (21.37s)
+
+Deploying services (azd deploy)
+
+  (✓) Done: Deploying service piglatinapp
+  - Endpoint: {{endpoint-url}}
+
+  Aspire Dashboard: {{dashboard-url}}
+```
+
+Finally, test your deployed Functions application using your favorite HTTP client:
+
+## [Unix](#tab/unix)
+
+```bash
+curl --request POST \
+  --url {{endpoint-url}}/api/Function1 \
+  --header 'Content-Type: application/json' \
+  --data '{
+  "value": "Welcome to Azure Functions"
+}'
+```
+
+## [Windows](#tab/windows)
+
+```powershell
+curl --request POST `
+  --url {{endpoint-url}}/api/Function1 `
+  --header 'Content-Type: application/json' `
+  --data '{
+  "value": "Welcome to Azure Functions"
+}'
+```
+
+---
+
+Support for Azure Functions in .NET Aspire is still in preview with support for a limited set of triggers including:
+
+- [HTTP triggers](/azure/azure-functions/functions-triggers-bindings?pivots=programming-language-csharp#supported-bindings)
+- [Azure Storage Queue triggers](/azure/azure-functions/functions-bindings-storage-queue?pivots=programming-language-csharp)
+- [Azure Storage Blob triggers](/azure/azure-functions/functions-bindings-storage-blob?pivots=programming-language-csharp)
+- [Azure Service Bus triggers](/azure/azure-functions/functions-bindings-service-bus?pivots=programming-language-csharp)
+- [Azure Event Hubs triggers](/azure/azure-functions/functions-bindings-event-hubs?pivots=programming-language-csharp)
+
+For the latest information on features support by the Azure Functions integration, see [the tracking issue](https://github.com/dotnet/aspire/issues/920).
+
+### Customization of Azure Container Apps
+
+One of the most requested features is the ability to customize the Azure Container Apps that are created by the app host without dropping to bicep. This is now possible by using the `PublishAsAzureContainerApp` method in `Aspire.Hosting.Azure.AppContainers` namespace. This method allows you to customize the Azure Container App definition that is created by the app host.
+
+Add the package reference to your project file:
```

Comment:
Do we need to tell people they need a new `azd`?

Reply:
How would they get it?